service/src/services/LibraryService.ts

```diff
@@ -39,8 +40,16 @@ export class LibraryService implements Service<fhir4.Library> {
     logger.debug(`Request Query: ${JSON.stringify(query, null, 2)}`);
     const parsedQuery = parseRequestSchema(query, LibrarySearchArgs);
     const mongoQuery = getMongoQueryFromRequest(parsedQuery);
-    const entries = await findResourcesWithQuery<fhir4.Library>(mongoQuery, 'Library');
-    return createSearchsetBundle(entries);
+
+    // if the _summary parameter with a value of count is included, then
+    // return a searchset bundle that excludes the entries
+    if (parsedQuery._summary && parsedQuery._summary === 'count') {
+      const count = await findResourceCountWithQuery(mongoQuery, 'Library');
+      return createSummarySearchsetBundle<fhir4.Library>(count);
+    } else {
```

Comment:
It looks like we default to a normal searchset bundle if the query includes the `_summary` parameter but the value isn't `count`. Do we want to throw an unsupported error for that case instead?

Reply:
Example: _summary=data

Reply:
Good call!